aries_cloudagent/resolver/default/indy.py

```diff
@@ -43,6 +46,60 @@ def supported_did_regex(self) -> Pattern:
         """Return supported_did_regex of Indy DID Resolver."""
         return IndyDID.PATTERN
 
+    def _add_endpoint_as_endpoint_value_pair(
+        self,
+        builder: DIDDocumentBuilder,
+        endpoint: str,
+        recipient_key: VerificationMethod,
+    ):
+        builder.service.add_didcomm(
+            ident=self.SERVICE_TYPE_DID_COMMUNICATION,
+            type_=self.SERVICE_TYPE_DID_COMMUNICATION,
+            service_endpoint=endpoint,
+            priority=1,
```

Comment:
This one sets priority to 1, other sets priority to 0 -- should probably be consistent.